tests/cases/fourslash/completionsPaths.ts

```diff
@@ -8,9 +8,12 @@
 // @Filename: /node_modules/x/bar.d.ts
 ////not read
 
-// @Filename: /a.ts
+// @Filename: /src/node_modules/y/index.d.ts
+////not read
+
+// @Filename: /src/a.ts
 ////import {} from "/*1*/";
 ////import {} from "x//*2*/";
 
```

Comment:
also suggest adding file /src/folder/b.ts and then adding same import statements there for the lookup